test/__snapshots__/loader.test.js.snap

```diff
@@ -1169,7 +1169,7 @@ exports[`loader should work with the "wrapper" options as an object notation: er
 exports[`loader should work with the "wrapper" options as an object notation: module 1`] = `
 "/*** IMPORTS FROM imports-loader ***/
 
-(function(myGlobalVariable, myOtherGlobalVariable) {
+(function() {
```

Comment:
Why it was changed? We should not change other tests, otherwise it is the breaking change

Reply:
Fixed it. It was a bug. 😅